docs/UserGuide.md

```diff
@@ -10,6 +10,65 @@ MediBase3 (MB3) is a **desktop app for doctors to manage their patients and appo
+
+**For experienced MediBase3 users:**
+1. If you need a quick refresher on how to use a specific feature in MediBase3, you can navigate to the [Features](#features) section.
+1. You may also skip to the [Command Summary](#command-summary) section for a quick overview of all the commands available in MediBase3 and their formats.
+
+For any additional information or queries, you can refer to the [FAQ](#faq) section or the [Known Issues](#known-issues) section.
+
+### Glossary
+As you read the user guide, you might encounter some unfamiliar technical terms. The table below provides the definitions for the key terms we will be using throughout the guide.
+
+Term | Definition
+---- | ----------
+Command |An input that is given to the application to perform a specific action. Commands are typically entered via the command box in the application.
+CLI | Command Line Interface(CLI) is a type of text-based interface that is used to interact with software via commands.
+GUI | Graphical User Interface(GUI) is a type of interface that allows users to interact with electronic devices through graphical icons and visual indicators.
+JAR | Java ARchive(JAR) is a package file format typically used to aggregate many Java class files and associated metadata and resources into one file for distribution.
+JSON | JavaScript Object Notation(JSON) is a lightweight data-interchange format that is easy for humans to read and write and easy for machines to parse and generate.
+Parameter | A parameter refers to a field that requires input from the user. For example, in the command `delete i/S1234567A`, `i/S1234567A` is a parameter.
+Prefix | A prefix is a keyword that is used to identify the type of parameter that follows it. For example, in the command `delete i/S1234567A`, `i/` is the prefix for the NRIC parameter.
+
+### Text Formatting Conventions
+We will be using different text formatting styles to help you better understand the content of the user guide.
+
+Format | Description
+------ | -----------
+[hyperlink](#how-to-use-this-user-guide) | Blue hyperlinks are used to bring you to external websites or another section of the User Guide for more information.
+**Bold** | Bold text is used to highlight important information or key points.
+`Monospace` | Text with a monospace font and a gray background is used to represent commands, parameters and code snippets.
+
+### Annotated Text-Box Conventions
+The following annotated text-boxes are used throught this guide to provide useful insights on MediBase3 and its features:
+
+{: .alert .alert-info}
+> :information_source: **Note:**
+> 
+> Provides additional information about MediBase3 that you should be aware of.
+
+{: .alert .alert-success}
+> :bulb: **Tip:**
+> 
+> Provides helpful tips or suggestions to improve your experience with MediBase3.
+
+{: .alert .alert-warning}
+> :exclamation: **Caution:**
+> 
+> Warns you about potential issues or errors that you might encounter while using MediBase3.
```

Comment:
There's a *link* alert box also, that the damith'anneh & co used previously  [over here](https://ay2425s1-cs2103t-f12-3.github.io/tp/Testing.html#running-tests)

```
{: .alert .alert-secondary}
:link: **Link**
```

Reply:
Ohh, we can consider using it, but will have to see where we can utilise it throughout the user guide.